package.json

```diff
@@ -23,6 +23,7 @@
   "version": "1.9.4",
   "dependencies": {
     "colorjs.io": "^0.5.2",
+    "espree": "^10.2.0",
```

Comment:
Is the idea to go with Espree (and manual traverse of AST) or are there plans to further investigate alternatives?

Reply:
Based on my own experiments, `Espree` is consistently faster than `Acorn`, but I just dug around and it seems that now `Espree` is built on top of `Acorn`, and extends `Acorn` for additional functionalities. I feel like maybe given larger files / over time, `Acorn` might be more efficient? Since what we require from a parser is pretty basis and not an extended capability.

And I totally forgot that we could traverse in a more efficient way! Regardless of which parser we choose, we can leverage on a traversal utility library, like`acorn-walk` for `acorn` or `estraverse` for `espree`.

Let me know what you think! I'm happy to do more formal benchmarking / experiments and have some sort of report / documentation before updating the implementation too.

cc: @davepagurek

Reply:
I think you can potentially do a bit of exploration for Espress vs Acorn, but the other things is whether using a linter is possible for this case or would manual AST traversal be the more realistic solution.